deployment/gce/lily/notifier_config.toml

```diff
@@ -0,0 +1,61 @@
+[API]
+  ListenAddress = "/ip4/127.0.0.1/tcp/1234/http"
+  RemoteListenAddress = ""
+  Timeout = "30s"
+
+[Backup]
+  DisableMetadataLog = false
+
+[Logging]
+
+[Libp2p]
+  ListenAddresses = ["/ip4/0.0.0.0/tcp/0", "/ip6/::/tcp/0"]
+  AnnounceAddresses = []
+  NoAnnounceAddresses = []
+  DisableNatPortMap = false
+  ConnMgrLow = 150
+  ConnMgrHigh = 180
+  ConnMgrGrace = "20s"
+
+[Pubsub]
+  Bootstrapper = false
+  RemoteTracer = "/dns4/pubsub-tracer.filecoin.io/tcp/4001/p2p/QmTd6UvR47vUidRNZ1ZKXHrAFhqTJAD27rKL9XYghEKgKX"
+
+[Client]
+  UseIpfs = false
+  IpfsOnlineMode = false
+  IpfsMAddr = ""
+  IpfsUseForRetrieval = false
+  SimultaneousTransfersForStorage = 20
+  SimultaneousTransfersForRetrieval = 20
+  OffChainRetrieval = false
+
+[Chainstore]
+  EnableSplitstore = false
+  [Chainstore.Splitstore]
+    ColdStoreType = ""
+    HotStoreType = ""
+    MarkSetType = ""
+    HotStoreMessageRetention = 0
+    HotStoreFullGCFrequency = 0
+    EnableColdStoreAutoPrune = false
+    ColdStoreFullGCFrequency = 0
+    ColdStoreRetention = 0
+
+[Storage]
```

Comment:
Similar to the worker the notifier will need access to the database. We will want to include a config for Postgres here, too.